src/plugins/README.md

```diff
@@ -0,0 +1,112 @@
+# Launcher Plugins
```

Comment:
We played with adding web search months ago, and one of the things was being able to have settings for search provider. I think for power users the plugin have it's own config file isn't bad, but for not-so-power users, should plugin authors have a way to surface settings? Or should plugins for now not need configuration?

Reply:
The meta file is just for describing information about the plugin to Pop Shell. Since these plugins are their own processes, they can do whatever they want as far as configs are concerned.